assets/js/util/scroll.js

```diff
@@ -81,24 +122,26 @@ export function getHeaderHeightWithoutNav( breakpoint ) {
 		'.react-joyride__overlay'
```

Comment:
Aha - thanks for pointing that out, I hadn't noticed we're explicitly setting the position to be non-sticky, and had assumed it was some DOM-related aspect, although hadn't dug into it (obviously!).

Given this is the case, I'd suggest we simply get the calculated style for the header and check whether the `position` property is `sticky`. I'm interested in your idea of using `scrollY` and `top` but that could give some indeterminate results (say both `scrollY` and `top` are `0`, we wouldn't know if the header is sticky or not).

Reply:
@aaemnnosttv, I do take your point, but I can't think of how we can determine the stickiness of the header purely based on its position in the viewport, or otherwise change the logic to avoid needing to make this determination.

Unless you've got any suggestions on how to achieve this, we may have no better alternative to checking the CSS `position` property...